cdap-docs/admin-manual/source/index.rst

```diff
@@ -96,26 +96,23 @@ installation and its security configuration.
 
 - |operations|_
 
-    .. |logging| replace:: **Logging:**
+    .. |logging| replace:: **Logging and Monitoring:**
     .. _logging: operations/logging.html
 
-    - |logging|_ Covers **CDAP support for logging** through standard SLF4J (Simple Logging Facade for Java) APIs.
+    - |logging|_ CDAP collects **logs and metrics** for all of its internal services, while CDAP itself
```

Comment:
It is not only for internal services. But also for applications. We do collect application logs and user metrics.